docs/_chapters/650-windows.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Tips for Windows users
+layout: default
+author: Fr Jeremy Krieg
+---
+
+Bnd and Bndtools runs on any system that can run Eclipse. However, every operating system is different and has its own quirks. Windows in particular has a few that impact developers using Bnd. While the Bnd experience may never be as smooth on Windows as it is on iOS or Linux, there are a few tweaks that make it a lot smoother than what comes out of the box.
```

Comment:
I think you mean `macOS` not `iOS` as `iOS` is the iPhone operating system.

Reply:
You mean you're not running Bndtools on your iPhone? 😄

Fixed.